### DISCUSSION TITLE
fatal error CVT1100:

### DISCUSSION THREAD

**Discussion by EHW-liao** [tauri-apps/tauri#7375] [Q&A]
I upgraded tauri to 1.4 and ran cargo run with fatal error CVT1100. How to solve this problem？

  = note: Non-UTF-8 output: CVTRES : fatal error CVT1100: \xd7\xca\xd4\xb4\xd6\xd8\xb8\xb4\xa1\xa3\xc0\xe0\xd0\xcd: VERSION\xa3\xac\xc3\xfb\xb3\xc6: 1\xa3\xac\xd3\xef\xd1\xd4: 0x0409\r\nLINK : fatal error LNK1123: \xd7\xaa\xbb\xbb\xb5\xbd COFF \xc6\xda\xbc\xe4\xca\xa7\xb0\xdc:
\xce\xc4\xbc\xfe\xce\xde\xd0\xa7\xbb\xf2\xcb\xf0\xbb\xb5\r\n

**Comment by FabianLars** [comment]
To be honest it's basically impossible to help you without a reproduction example here. That error could stem from virtually anywhere.

Also, if you just upgraded, first make sure you really updated _all_ of tauri's deps, so tauri and tauri-build in cargo.toml (run cargo update afterwards) and the cli (`@tauri-apps/cli` in package.json or `tauri-cli` via cargo, depending on what you use). Lastly, delete the `target/` dir and try again.

**Comment by EHW-liao** [comment]
你好你的邮件我已收到。                                             廖海鹏
